help/develop/domains/chem/cheminformatics.md

```diff
@@ -31,59 +31,36 @@ descriptors(table, column, descriptors);
 ```
 
 The three first functions, `searchSubstructure`, `getSimilarities` and `findSimilar`, are currently made client-based,
-using [RDKit-JS](https://github.com/rdkit/RDKitjs) library. We are planning to support both server-side and browser-side
+using [RDKit JS](https://github.com/rdkit/rdkit/blob/master/Code/MinimalLib/minilib.h). We are planning to support both server-side and browser-side
 modes for these functions in the future, while the API remains as described below.
 
 ### Substructure search
 
 `searchSubstructure(column, pattern = null, settings = { substructLibrary: true })`
 
-This function performs substructure search using RDKit, and returns a result as a [BitSet](../packages/js-api.md#bitset),
-where the i-th bit is set to 1 when the i-th element of the input `column` contains a substructure given by a `pattern`,
-and all other bits there are set to 0.
+This function performs substructure search using RDKit JS. It returns a [BitSet](../packages/js-api.md#bitset).
+If i-th element of the input `column` contains a substructure given by a `pattern` the i-th bit set to 1, otherwise the bit is set to 0.
 
-`column` is a column of type String containing molecules, each in any notation
-RDKit-JS [supports](https://github.com/rdkit/rdkit/blob/master/Code/MinimalLib/minilib.h): smiles, cxsmiles, molblock,
-v3Kmolblock, and inchi. Same applies to a `pattern`: this can be a string representation of a molecule in any of the
-previous notations.
+`column` is a column of type String. It contains molecules in any notation [supported by RDKit JS](https://github.com/rdkit/rdkit/blob/master/Code/MinimalLib/minilib.h): smiles, cxsmiles, molblock, v3Kmolblock, and inchi. Same applies to a `pattern`: this is a string representation of a molecule in any of the previous notations.
```

Comment:
`column is a column of type String` sounds weird to me. 
Isn't it better to write something like this:
`column stands for [column](https://datagrok.ai/js-api/classes/dg.Column) that contains molecules in any notation.`

Reply:
Done